src/components/primitives/Image/index.tsx

```diff
@@ -28,7 +28,8 @@ const Image = (props: IImageProps, ref: any) => {
       finalSource.current = { uri: src };
     }
     return finalSource.current;
-  }, [source, src]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Since setting `source` in useEffects' deps is causing an error, I am using `source.uri` and disabling `react-hooks/exhaustive-deps` for `source`